src/redux/actions/portfolio-actions.js

```diff
@@ -107,3 +108,49 @@ export const selectPortfolioItem = (portfolioItem) => ({
   payload: portfolioItem
 });
 
+export const queryPortfolio = apiProps => ({
+  type: ActionTypes.QUERY_PORTFOLIO,
+  payload: ShareHelper.queryPortfolio(apiProps).then(({ data }) => data)
+});
+
+export const sharePortfolio = (portfolioData) => ({
+  type: ActionTypes.SHARE_PORTFOLIO,
+  payload: ShareHelper.sharePortfolio({
+    ...portfolioData
+  }),
+  meta: {
+    notifications: {
+      fulfilled: {
+        variant: 'success',
+        title: 'Success updating portfolio',
+        description: 'The portfolio was shared successfully.'
+      },
+      rejected: {
```

Comment:
I would prefer utilizing the default notification mechanism for rejected responses for now because it shows the actual server error. I would guess that eventually the APIs will return some reasonable error messages that can be displayed to users.

Reply:
this function was moved to the share-actions.js file - I will make this change over in that file.